apps/root/package.json

```diff
@@ -84,7 +84,7 @@
   },
   "dependencies": {
     "@apollo/client": "^3.3.20",
-    "@arcxmoney/analytics": "^2.0.2",
+    "@arcxmoney/analytics": "^2.0.3",
```

Comment:
This upgrade should not be necessary right?

Reply:
no need really, but it threw an arcx-related error, so I also took the opportunity to update the package :)